### DISCUSSION TITLE
Browser not opening when testing notification

### DISCUSSION THREAD

**Discussion by ThinkSalat** [jef/streetmerchant#1128] [Help]
Here's my .env

AUTO_ADD_TO_CART="true"
OPEN_BROWSER="true"
DESKTOP_NOTIFICATIONS="true"
LOW_BANDWIDTH=""
MAX_PRICE_SERIES_3080="750"
OPEN_BROWSER="true"
SHOW_ONLY_BRANDS="nvidia"
SHOW_ONLY_MODELS="founders edition"
SHOW_ONLY_SERIES="3080"
STORES="nvidia, bestbuy, amazon, newegg"
USER_AGENT="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36"

For some reason it's not opening my browser when I run `npm run test:notification`, but the desktop notification is showing up. The issue with the desktop notification is that it's not actually taking me to the link when I click on it.

**Comment by Niswendel** [comment]
that test is only for notifications, i recomend scanniong for every item on a bunch of stores for like an hour and then you can test everything.
